pkg/controller/gslb/gslb_controller_test.go

```diff
@@ -631,6 +625,20 @@ func TestGslbController(t *testing.T) {
 			t.Errorf("got:\n %s Gslb ingress annotations,\n\n want:\n %s", ingress.Annotations, expectedAnnotations)
 		}
 	})
+
+	t.Run("Reflect GeoTag in the Status", func(t *testing.T) {
+		err = os.Setenv("CLUSTER_GEO_TAG", "eu")
```

Comment:
Shouldn't we unset `CLUSTER_GEO_TAG` at the end of the test?

Reply:
@kuritka good point, will amend, thanks